lambda_src/drivers/process_sentry_logger.py

```diff
@@ -77,16 +77,17 @@ def process_row(
     """Each row is sent to Sentry via the SENTRY_DRIVER_LOGGER
 
     Args:
-        error (str): The actual error message.
-        query_id (str): The query ID of the failed query.
+        account (str): The Snowflake account name.
+        region (str): Region of the Snowflake account.
         database (str): The DB.
         schema (str): The schema.
         name (str): The object name.
-        ts (str): The timestamp of the error.
         history_type (str): pipe or task or query.
-        region (str): Region of the Snowflake account.
-        account (str): The Snowflake account name.
+        error (str): The actual error message.
+        ts (str): The timestamp of the error.
     """
+    CONSOLE_LOGGER.info(f'sentry_logger driver invoked.')
```

Comment:
what's the logging strategy here? is this something we want to log on every driver? if so, can we put it into the invoking code, and make it debug rather than info?

Reply:
debug level makes sense. However, for the first part we don't know for sure until it's actually invoked. Invoking code just calls the `process_row` function of the respective driver. We still want to know from the drivers PoV that the correct driver was invoked. From invoker side of the code, we can have a log saying `Invoking the {driver_name} driver.`. Is the preferable?

Does that answer the strategy question? When I was looking into the logs, I didn't know if the sentry_logger driver was correctly invoked and wanted the logs to reflect that.

Reply:
Will add invoking (from lambda_function) and invoked (from respective) for every driver in a separate PR.